### DISCUSSION TITLE
Phpstrom's copilot can't log in to GitHub at all!

### DISCUSSION THREAD

**Discussion by peter-dou** [community/community#16515] [Copilot Conversations] [answered]
Phpstrom's copilot can't log in to GitHub at all! Not without authorization, other accounts have been authorized

**Comment by nyxo-dev** [comment]
#16501 this account has been authorized!Don't end for no reason

**Comment by jansorg** [comment]
@peter-dou Are you seeing an error message? Are you perhaps using a proxy server?

**Reply by nyxo-dev** [reply]
@Peter Dou and I are alone, I don't see the error message! Proxy server has also been used, but the problem still exists!

**Comment by davecheney** [comment, marked as answer]
@peter-dou please see https://github.com/github/feedback/discussions/16501

**Reply by nyxo-dev** [reply]
My account has been authorized! The current problem is that you can't pull up the login GitHub

**Reply by davecheney** [reply]
@isjinlongw I see no telemetry from your client. Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by nyxo-dev** [reply]
@davecheney I can log in a few days ago! I can't log in suddenly since yesterday!

**Reply by nyxo-dev** [reply]
@davecheney I downloaded the copilot of vscode and logged in to GitHub without any problem
